### PR TITLE
Fix table query issue of not working with conditional blocks

### PR DESCRIPTION
**Pull request by anupama-pathirage**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: tests/ballerina-unit-test/src/test/resources/test-src/types/table/table_sql.bal [hunk 1]
```diff
@@ -285,3 +285,28 @@ function testTableToString() returns table<record {}> {
...
+
+    _ = personTable.add(p1);
+    _ = personTable.add(p2);
+    _ = personTable.add(p3);
+    _ = personTable.add(p4);
+
+    if(recordCount == 0) {
+        table<Person> personTableCopy = from personTable select * limit 2;
+        while (personTableCopy.hasNext()) {
+            var rs = personTableCopy.getNext();
```
Comment (MaryamZi) [inline tests/ballerina-unit-test/src/test/resources/test-src/types/table/table_sql.bal]: We can ignore the return value right? We don't seem to be using `rs`?

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableSqlQueryTest.java [hunk 2]
```diff
@@ -89,6 +89,13 @@ public void testSimpleSelectAllWithLimit() {
         Assert.assertEquals(((BInteger) returns[0]).intValue(), 2);
     }
 
+    @Test(groups = "TableQueryTest", description = "Do a simple select within a if block")
```
Comment (NatashaWso2) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableSqlQueryTest.java]:
```suggestion
    @Test(groups = "TableQueryTest", description = "Do a simple select within an if block")
```

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableSqlQueryTest.java [hunk 3]
```diff
@@ -89,6 +89,13 @@ public void testSimpleSelectAllWithLimit() {
...
+    public void testSimpleSelectAllWithCondition() {
+        BValue[] args = {};
+        BValue[] returns = BRunUtil.invoke(result, "testSimpleSelectAllWithCondition", args);
```
Comment (NatashaWso2) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableSqlQueryTest.java]:
```suggestion
        BValue[] returns = BRunUtil.invoke(result, "testSimpleSelectAllWithCondition", new BValue[0]);
```

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableSqlQueryTest.java [hunk 4]
```diff
@@ -89,6 +89,13 @@ public void testSimpleSelectAllWithLimit() {
         Assert.assertEquals(((BInteger) returns[0]).intValue(), 2);
     }
 
+    @Test(groups = "TableQueryTest", description = "Do a simple select within a if block")
+    public void testSimpleSelectAllWithCondition() {
+        BValue[] args = {};
```
Comment (NatashaWso2) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableSqlQueryTest.java]:
```suggestion
```

File: tests/ballerina-unit-test/src/test/resources/test-src/types/table/table_sql.bal [hunk 5]
```diff
@@ -285,3 +285,28 @@ function testTableToString() returns table<record {}> {
 
     return personTableCopy;
 }
+
+
```
Comment (NatashaWso2) [inline tests/ballerina-unit-test/src/test/resources/test-src/types/table/table_sql.bal]:
```suggestion
```

**Pull request merged**